app/controllers/miq_ae_customization_controller.rb

```diff
@@ -481,7 +480,7 @@ def setup_presenter_for_old_dialogs_tree(nodetype, presenter)
                 else
                   ""
                 end
-        @right_cell_text = _("Editing %{model} \"%{name}\"") % {:name => @dialog.description, :model => "#{title} #{ui_lookup(:model => "MiqDialog")}"}
+        @right_cell_text = _("Editing %{title} Dialog \"%{name}\"") % {:name => @dialog.description, :title => title}
```

Comment:
Probably a bug here .. `if @edit && params[:typ] != "copy", this will be `Editing Editing  Dialog ...`.
And `Editing Copy  Dialog` doesn't make much sense either..